modules/appeals_api/app/models/concerns/appeals_api/appeal_scopes.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module AppealScopes
+    extend ActiveSupport::Concern
+
+    included do
+      scope :without_status_updates_since, lambda { |time|
+        status_update_table = AppealsApi::StatusUpdate.table_name
+        join_clause = <<~JOIN
+          LEFT JOIN #{status_update_table}
+          ON #{table_name}.id = CAST(#{status_update_table}.statusable_id as uuid)
```

Comment:
This `CAST` is necessary because the StatusUpdate table's `statusable_id` column is a varchar, unlike the appeal `id` columns, which are uuids. This is also why I'm not using the `joins(:status_udpates)` syntax here.

I think it would be a better idea to just update the `statusable_id` column to be a uuid instead of a varchar permanently, but when adding a migration like this:
```ruby
class ChangeStatusUpdateStatusableIdToUuid < ActiveRecord::Migration[7.1]
  def change
    change_column :status_updates, :statusable_id, :uuid
  end
end
```
...the strong_migrations gem lets us know that this change would be more involved than it seems:
```
=== Dangerous operation detected #strong_migrations ===

Changing the type of an existing column blocks reads and writes
while the entire table is rewritten. A safer approach is to:

1. Create a new column
2. Write to both columns
3. Backfill data from the old column to the new column
4. Move reads from the old column to the new column
5. Stop writing to the old column
6. Drop the old column
```

I decided to go with the `CAST` here instead because we're under a lot of time pressure right now.

Reply:
Thanks for the explanation and for finding a way to make this complex query work.